src/site/layout/javascript/artifacts.rs

```diff
@@ -139,19 +139,20 @@ window.os = os;
 let hit = Array.from(document.querySelectorAll(".target[data-targets]")).find(
   (a) => a.attributes["data-targets"].value.includes(os)
 );
+let backupButton = document.querySelector(".backup-download");
```

Comment:
what does "backup" mean here

Reply:
It's the button that shows up on android for example if nothing is matched

Reply:
ahhhhhhh

Reply:
i think it would be helpful to rename this to "undetectable" or something like that

Reply:
if I change the classname we have to make a new fringe release

Reply:
Danke